pkg/services/serviceaccounts/database/database.go

```diff
@@ -130,7 +129,7 @@ func (s *ServiceAccountsStoreImpl) ListTokens(ctx context.Context, orgID int64,
 
 		sess = dbSession.Limit(100, 0).
 			Join("inner", "user", "user.id = api_key.service_account_id").
-			Where("user.org_id=? AND user.id=? AND ( expires IS NULL or expires >= ?)", orgID, serviceAccountID, time.Now().Unix()).
+			Where("user.org_id=? AND user.id=?", orgID, serviceAccountID).
```

Comment:
remove the filtering in the sql query for now, so that we filter disabled in the frontend eventually.